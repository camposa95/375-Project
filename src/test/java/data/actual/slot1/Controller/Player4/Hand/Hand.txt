LUMBER: 2
WOOL: 0
BRICK: 0
GRAIN: 1
ORE: 0
