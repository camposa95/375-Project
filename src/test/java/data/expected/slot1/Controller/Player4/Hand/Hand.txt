LUMBER: 2
GRAIN: 1
WOOL: 0
BRICK: 0
ORE: 0
